shard.yml

```diff
@@ -1,13 +1,12 @@
 name: invidious
-version: 0.20.1
+version: 2.20240825.2-dev
 
 authors:
   - Omar Roth <omarroth@protonmail.com>
-  - Invidious team
+  - Invidious team <contact@invidious.io>
```

Comment:
Regarding the authors section should we change it to only include the Invidious team as to reflect the current authors?

Reply:
Why not, though `Invidious team and contributors` is more representative of all the people that worked on invidious!